__init__.py

```diff
@@ -424,6 +424,27 @@ def __init__(self, cookie_file=None, domain_name="", key_file=None):
 
         super().__init__(browser='Opera', cookie_file=cookie_file, domain_name=domain_name, key_file=key_file, **args)
 
+class Brave(ChromiumBased):
+    def __init__(self, cookie_file=None, domain_name="", key_file=None):
+        args = {
+            'linux_cookies':['~/.config/BraveSoftware/Brave-Browser/Default/Cookies'],
+            'windows_cookies':[
+                    {'env':'APPDATA', 'path':'..\\Local\\BraveSoftware\\Brave-Browser\\User Data\\Default\\Cookies'},
+                    {'env':'LOCALAPPDATA', 'path':'BraveSoftware\\Brave-Browser\\User Data\\Default\\Cookies'},
+                    {'env':'APPDATA', 'path':'BraveSoftware\\Brave-Browser\\User Data\\Default\\Cookies'}
+            ],
+            'osx_cookies': ['~/Library/Application Support/BraveSoftware/Brave-Browser/Default/Cookies'],
+            'windows_keys': [
+                    {'env':'APPDATA', 'path':'..\\Local\\BraveSoftware\\Brave-Browser\\User Data\\Local State'},
+                    {'env':'LOCALAPPDATA', 'path':'BraveSoftware\\Brave-Browser\\User Data\\Local State'},
+                    {'env':'APPDATA', 'path':'BraveSoftware\\Brave-Browser\\User Data\\Local State'}
+            ],
+            'os_crypt_name':'chromium',
```

Comment:
@bjlange os_crypt_name for Brave Browser is `brave`. Other than that everything works perfectly. Tested on Linux and Windows.